feature/search/src/main/java/com/google/samples/apps/nowinandroid/feature/search/SearchScreen.kt

```diff
@@ -312,11 +312,7 @@ private fun SearchResultBody(
             state = state,
         ) {
             if (topics.isNotEmpty()) {
-                item(
-                    span = {
-                        GridItemSpan(maxLineSpan)
-                    },
-                ) {
+                item(span = { GridItemSpan(maxLineSpan) }) {
```

Comment:
For consistency, the `span` argument should probably be on the next line

Reply:
I solved this problem, thank you very much